esx_service/vmdk_ops.py

```diff
@@ -696,20 +696,24 @@ def get_vol_path(datastore, tenant_name=None):
     # is created on <datastore>/DOCK_VOLS_DIR/tenant_uuid
     # a symlink <datastore>/DOCK_VOLS_DIR/tenant_name will be created to point to
     # path <datastore>/DOCK_VOLS_DIR/tenant_uuid
+    # If the command is under a tenant then the path returned contains tenant
+    # name not UUID. This is to make logs more readable. OS will resolve this path
+    # as a symlink with tenant_name is already present.
 
-    path = dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
+    rpath = path = dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
```

Comment:
what does "r" stands for in "rpath" ?  It would be better to use full names, whatever it is .... - easier to read